Hackney.Core/DynamoDb/Converters/DynamoDbEnumConverter.cs

```diff
@@ -19,9 +19,10 @@ public DynamoDBEntry ToEntry(object value)
         public object FromEntry(DynamoDBEntry entry)
         {
             Primitive primitive = entry as Primitive;
-            if (null == primitive) return default(TEnum);
+            var entryStringValue = primitive?.AsString();
+            if (string.IsNullOrEmpty(entryStringValue)) return default(TEnum);
```

Comment:
wouldn't this just return the first enum?

Reply:
Yes... that is the default behaviour if no value is supplied, and what is currently supplied if it is null in the json.
If you want an enum property to be able to be null, then you can make your property nullable.